open-metadata-implementation/access-services/cognos/cognos-spring/src/main/java/org/odpi/openmetadata/accessservices/cognos/server/spring/CognosOMASResource.java

```diff
@@ -0,0 +1,110 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.cognos.server.spring;
+
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.odpi.openmetadata.accessservices.cognos.responses.CognosOMASAPIResponse;
+import org.odpi.openmetadata.accessservices.cognos.server.CognosRestServices;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+@RestController
+@RequestMapping("/servers/{serverName}/open-metadata/access-services/cognos/users/{userId}/")
+
+@Tag(name="Cognos OMAS",
+	description="The Cognos OMAS queries RDBMS metadata and creates modules for IBM Cognos Analytics.",
+	externalDocs=@ExternalDocumentation(description="Cognos Open Metadata Access Service (OMAS)",
+	url="https://egeria.odpi.org/open-metadata-implementation/access-services/cognos/"))
+
+
+public class CognosOMASResource {
+
+    private final CognosRestServices restAPI = new CognosRestServices();
+
+	/**
+	 * Get list of databases.
+     * @param serverName  unique identifier for requested server.
+     * @param userId      the unique identifier for the user
+	 * @return Cognos response contains list of databases.
+	 */
+	@GetMapping(path = "/databases")
+	public CognosOMASAPIResponse getDatabases(
+			@PathVariable("serverName") String serverName,
+            @PathVariable("userId") String userId
+			)
+	{
+		return restAPI.getDatabases(serverName);
+	}
+
+	/**
+	 * Get list of schemas of a given dataSource.
+     * @param serverName  unique identifier for requested server.
+     * @param userId      the unique identifier for the user
+	 * @param dataSource data source GUID.
+	 * @return Cognos response contains list of database schemas.
+	 */
+	@GetMapping(path = "/{dataSourceGUID}/schemas")
+	public CognosOMASAPIResponse getSchemas(
```

Comment:
Need a specialized response object for each type of response - otherwise can not see what is returned.